Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/SimpleShapeMonteCarloAbsorptionTest.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.simpleapi import (SimpleShapeMonteCarloAbsorption, Load, ConvertUnits,
+                              CompareWorkspaces, SetSampleMaterial, CreateSampleWorkspace,
+                              DeleteWorkspace)
+from mantid.kernel import *
+from mantid.api import *
+
+import unittest
+
+
+class SimpleShapeMonteCarloAbsorptionTest(unittest.TestCase):
+    def setUp(self):
+        """
+
+        """
+        red_ws = Load('irs26176_graphite002_red.nxs')
```

Comment:
Should we have one more test with ILL reduced data? Just to make sure everything is and stays consistent.